cmobase/introduction.adoc

```diff
@@ -1,9 +1,9 @@
 [#intro,reftext="Introduction"]
 == Introduction
 
-_Cache management operation_ (or _CMO_) instructions perform operations on
-cached copies of data in the memory hierarchy. These instructions are grouped by
-operation into the following instruction classes:
+_Cache management operation_ (or _CMO_) instructions are intended to perform
```

Comment:
Not quite following the reason to add "are intended to" to the sentence.

Reply:
If CBO.ZERO instructions are allowed to succeed against uncacheable or non-cached address ranges, then the original sentence, as written, is false.  I think we all agree that the primary use case for CBO.ZERO is against cached memory, however.

Reply:
Got it. I will work on a more inclusive statement for this paragraph (and the one introducing CBOs).

Reply:
Done in working draft